src/rpdk/guard_rail/core/stateful.py

```diff
@@ -89,16 +91,59 @@ class DIFFKEYS:
 }
 
 
+def remove_key(document: Dict[str, Any], key: str):
+    """removes only top level property"""
+    cloned_document = deepcopy(document)
+    if key in cloned_document:
+        del cloned_document[key]
+    return cloned_document
+
+
+def get_value(document: Dict[str, Any], key: str):
```

Comment:
nit: is this function really required? It does nothing else but `<dict>.get()`

Reply:
yea you right